flask_rebar/swagger_generation/generator_utils.py

```diff
@@ -10,13 +10,32 @@
 import copy
 import re
 from collections import namedtuple, OrderedDict
-
+from typing import (
+    overload,
+    Any,
+    Callable,
+    Dict,
+    Iterator,
+    List,
+    Optional,
+    Tuple,
+    Set,
+    Union,
+    TYPE_CHECKING,
+)
+
+from flask_rebar.authenticators import Authenticator
 from flask_rebar.utils.defaults import USE_DEFAULT
 from flask_rebar.swagger_generation import swagger_words as sw
 from flask_rebar.swagger_generation.marshmallow_to_swagger import get_swagger_title
+from marshmallow import Schema
+
+if TYPE_CHECKING:
+    from flask_rebar.rebar import HandlerRegistry
```

Comment:
rebar also imports from files in swagger_generation/ so this is needed to avoid circular imports